bCNC/Sender.py

```diff
@@ -860,7 +861,9 @@ def serialIO(self):
 
                 tosend = None
                 if not self.running and t - tg > G_POLL:
-                    tosend = b"$G\n"  # FIXME: move to controller specific class
-                    sline.append(tosend)
-                    cline.append(len(tosend))
+                    #tosend = b"$G\n"  # FIXME: move to controller specific class
```

Comment:
It's a small detail, but these three commented out lines seem like they could be removed. :smile:

Reply:
Absolutely.  I wanted to leave some evidence for discussion and others testing.  I *think* it shouldn't affect any other platforms ...  famous last words.

Thanks!